R/zzz.R

```diff
@@ -100,6 +100,52 @@
   }
 }
 
+.onAttach <- function(libname, pkgname) {
+  if (is_positron()) {
+    # on Positron we want to enable the reticulate integration.
+    tryCatch({
+      maybe_enable_positron_reticulate_integration()
+    }, error = function(e) {
+      # if we fail to enable, we don't want to throw any errors.
+      # it's likely running in a incompatible Positron version
+    })
+  }
+}
+
+maybe_enable_positron_reticulate_integration <- function() {
+  enabled <- .ps.ui.evaluateWhenClause("config.positron.reticulate.enabled")
+  if (!enabled) {
+    # when not enabled, we check if the user has explicitly disabled or if
+    # it's just set false by default.
+    explicitly_disabled <- tryCatch({
+      .ps.ui.executeCommand("positron.reticulate.isEnabledExplicitlySet")
+    }, error = function(e) {
+      # if error, the command is not available, it's likely that we are running an old Positron version
+      # so we treat it as explicily disabled, as we won't be able to turn it on.
+      TRUE 
+    })
+
+    if (!explicitly_disabled) {
+      tryCatch({
+        .ps.ui.executeCommand("positron.reticulate.toggleEnabled")
+      
+        if (requireNamespace("cli", quietly = TRUE)) {
+          cli::cli_inform(
+            "Reticulate integration has been enabled, to disable set {.url positron://settings/positron.reticulate.enabled} to {.val false}.", 
```

Comment:
```suggestion
            "Positron's reticulate integration is now enabled. To disable, toggle {.href [positron.reticulate](positron://settings/positron.reticulate.enabled)}", 
```

Clicking the links brings me to a GUI checkbox to disable, rather than the source-mode json. Tweaked wording a little to make clear it's a Positron feature and present the link differently.